### ISSUE TITLE
Staging Review finding: Focus is lost when submitting the form

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#104552]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** BAH-MBS-SelfServ
**Product:** Current - Pre-need Burial Requested change to Pre-Need Self Service
**Feature:** Pre-Need modernization and CaMEO Integration

## Findings details
**VA.gov Experience Standard - issue:** [User experiences predictable interactions with components and patterns.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When you submit the form and go to the confirmation page, keyboard focus is lost. In this particular case the biggest impact is on screen reader users, since keyboard focus often determines what will be announced by the screen reader during page transitions in an SPA. Right now after you press submit, there's nothing announced by screen readers --- which means if you don't have any visual cues for the page change, you have to guess about what to do next. Did it work? Do I need to press submit again? Have I waited long enough? etc.

<details><summary>Link, screenshot or steps to recreate</summary>

[video]

</details>

## Recommended action

When transitioning to the confirmation page, focus should go to the "You’ve submitted your application" text, with `tabindex="-1"` to make it programmatically focusable.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by MichelleDieudonne** [comment]
We have created a bug for this in our internal tracking system and will be pulling this in to work in the next few sprints. https://jira.devops.va.gov/browse/MBMS-88748
